homeassistant/components/tuya/light.py

```diff
@@ -401,7 +398,7 @@ def __init__(
         super().__init__(device, device_manager)
         self.entity_description = description
         self._attr_unique_id = f"{super().unique_id}{description.key}"
-        self._attr_supported_color_modes = {COLOR_MODE_ONOFF}
+        self._attr_supported_color_modes = {ColorMode.ONOFF}
```

Comment:
The docs say we shouldn't add ONOFF unless no other color modes have been added

https://developers.home-assistant.io/docs/core/entity/light?_highlight=color#color-modes

Reply:
I have adjusted the logic to start with an empty set, and only add ONOFF if it is still empty after initialisation.